core/deviceadaptor.cpp

```diff
@@ -120,12 +120,10 @@ RingBufferBase* DeviceAdaptor::findBuffer(const QString& name) const
 bool DeviceAdaptor::setStandbyOverride(bool override)
 {
     standbyOverride_ = override;
-    if (screenBlanked_) {
-        if (override) {
-            resume();
-        } else {
-            standby();
-        }
+    if (override) {
+        resume();
+    } else {
+        standby();
```

Comment:
I probably do not understand what happens here, but ...

It looks like the sensor is stopped if: standby override is removed -> which would not make sense if the device is in a state where override is not needed for the sensor to stay enabled.

I would have expected something like:

    if(override) {resume();} else if(_screenBlanked) {standby();} else { /* no change */ }